lib/ui/view/channel/channel_view/channel_page_view.dart

```diff
@@ -176,7 +179,10 @@ AppBar appBar(String text, String nexttext, BuildContext context) {
       Padding(
         padding: const EdgeInsets.only(right: 20.0),
         child: GestureDetector(
-          onTap: () {},
+          onTap: () {
+            Navigator.of(context).push(
```

Comment:
Kindly remove this logic from the UI and make use of `model.navigateToChannelView()`